src/components/PositionListItem/index.tsx

```diff
@@ -7,54 +7,59 @@ import DoubleCurrencyLogo from 'components/DoubleLogo'
 import HoverInlineText from 'components/HoverInlineText'
 import Loader from 'components/Loader'
 import { RowBetween } from 'components/Row'
+import { RedesignVariant, useRedesignFlag } from 'featureFlags/flags/redesign'
 import { useToken } from 'hooks/Tokens'
 import useIsTickAtLimit from 'hooks/useIsTickAtLimit'
 import { usePool } from 'hooks/usePools'
 import { useMemo } from 'react'
 import { Link } from 'react-router-dom'
 import { Bound } from 'state/mint/v3/actions'
-import styled from 'styled-components/macro'
+import styled, { css } from 'styled-components/macro'
 import { HideSmall, MEDIA_WIDTHS, SmallOnly } from 'theme'
 import { PositionDetails } from 'types/position'
 import { formatTickPrice } from 'utils/formatTickPrice'
 import { unwrappedToken } from 'utils/unwrappedToken'
 
 import { DAI, USDC_MAINNET, USDT, WBTC, WRAPPED_NATIVE_CURRENCY } from '../../constants/tokens'
 
-const LinkRow = styled(Link)`
+const LinkRow = styled(Link)<{ redesignFlag: boolean }>`
   align-items: center;
-  border-radius: 20px;
   display: flex;
   cursor: pointer;
   user-select: none;
   display: flex;
   flex-direction: column;
-
+  border-radius: ${({ redesignFlag }) => (redesignFlag ? '0px' : '20px')};
   justify-content: space-between;
   color: ${({ theme }) => theme.deprecated_text1};
-  margin: 8px 0;
+  margin: ${({ redesignFlag }) => (redesignFlag ? '0px' : '8px 0')};
   padding: 16px;
   text-decoration: none;
   font-weight: 500;
-  background-color: ${({ theme }) => theme.deprecated_bg1};
+
+  ${({ redesignFlag, theme }) =>
+    !redesignFlag &&
+    css`
+      background-color: ${theme.deprecated_bg1};
+    `}
 
   &:last-of-type {
-    margin: 8px 0 0 0;
+    margin: ${({ redesignFlag }) => (redesignFlag ? '0px' : '8px 0 0 0')};
   }
   & > div:not(:first-child) {
     text-align: center;
   }
   :hover {
-    background-color: ${({ theme }) => theme.deprecated_bg2};
+    background-color: ${({ theme, redesignFlag }) => (redesignFlag ? theme.hoverDefault : theme.deprecated_bg2)};
   }
 
   @media screen and (min-width: ${MEDIA_WIDTHS.deprecated_upToSmall}px) {
     /* flex-direction: row; */
   }
 
-  ${({ theme }) => theme.deprecated_mediaWidth.deprecated_upToSmall`
+  ${({ theme, redesignFlag }) => theme.deprecated_mediaWidth.deprecated_upToSmall`
     flex-direction: column;
-    row-gap: 12px;
+    ${redesignFlag && `row-gap: 8px;`}
```

Comment:
nit: use `css` template here, or use ' or " instead of nesting ` inside another template string